__tests__/articlesRoutes.tests.js

```diff
@@ -47,3 +47,54 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("GET /api/articles", () => {
+  describe("Happy path", () => {
+    it("should return a 200", async () => {
+      await request(app).get("/api/articles").expect(200);
+    });
+    it("should return an array of objects", async () => {
```

Comment:
is that what you're testing here? You've just tested to see if it's an array with length greater than 0

Reply:
Agreed. Will update description